server/pregnancy-centers/schema/helpers.js

```diff
@@ -0,0 +1,12 @@
+const _ = require('lodash')
+
+const pregnancyCenterServices = require('../pregnancy-center-services')
+
+/*
+	Create the Joi and Mongoose schemas based off of the pregnancy-center-services
+	array so that services only need to be updated in one place.
+*/
+exports.getPregnancyCenterServicesSchema = dataType => {
+	const services = _.mapKeys(pregnancyCenterServices, service => service.id)
+	return _.mapValues(services, () => dataType)
```

Comment:
This is using more es6 and functional programming than we've been using thus far. I know you've been getting up to speed quickly on JS, but if you would like me to explain what's going on here, I would be happy to do so!! Just let me know.